### ISSUE TITLE
[Feature]: Do not switch the OBS scene via M³ while media is playing

### ISSUE THREAD

**Issue by fschein** [sircharlo/meeting-media-manager#3232]
### Is your feature request related to a problem?

In the previous version of M³, when selecting another scene directly from M³, the scene didn’t switch immediately. I used it as "next scene when the video finishes." In the current version, M³ switches the scene immediately.

### Describe the solution you'd like

If I want to switch the scene immediately, I can use OBS. If I want the scene to change after the video finishes, I can select the next scene in M³, just like in the previous version.

### Describe alternatives you've considered

One alternative could be adding an option or checkbox in M³ settings that lets the user decide whether the scene should switch immediately or only after the video ends. This way, users can configure the behavior based on their needs.

**Comment by mtdvlpr** [comment]
Duplicate of #3214

**Comment by ux-git** [comment]
@mtdvlpr The "Switch scene immediately or only after the video ends" setting sounds pretty good though

**Comment by fschein** [comment]
Thank you for addressing this issue. I understand that this behavior was discussed previously, and I appreciate the feedback. However, I would like to suggest that adding an option to control scene switching behavior within M³ could be beneficial for many users. While the current setup is fine for those who prefer immediate scene switching, there are users, including myself, who found the previous behavior (where scenes would switch only after the media finishes) more useful for certain situations. Thank you for considering this enhancement!

**Comment by sircharlo** [comment]
In such a case that we add this setting and it's enabled by the user, I suppose the media scene would no longer be displayed on the user in the popup? The popup would serve as a way to change the scene immediately if no media is playing, or after media is stopped, correct?

**Comment by mtdvlpr** [comment]
Yeah, it wouldn't make sense to show the media scene as a choice when it's always enforced during media.

**Comment by sircharlo** [comment]
> Yeah, it wouldn't make sense to show the media scene as a choice when it's always enforced during media.

My thoughts exactly

**Comment by fschein** [comment]
This issue can be considered low priority. I miss the feature, but it doesn't require effort at the moment.

**Comment by sircharlo** [comment]
Great news, @mtdvlpr added this feature for you!

![image](https://github.com/user-attachments/assets/0a8e0c85-1090-4b2f-9716-0f16135b0e0c)

**Comment by ux-git** [comment]
@sircharlo what a real brotherly love 😊😄

**Comment by fschein** [comment]
I'm speechless. Thank you so much 😊
